tests/hotplug/cpu.go

```diff
@@ -60,6 +60,23 @@ var _ = Describe("[sig-compute][Serial]CPU Hotplug", decorators.SigCompute, deco
 
 	})
 
+	Context("with requests without topology", func() {
+
+		It("should be able to start", func() {
+
+			vmi := libvmifact.NewAlpineWithTestTooling(
+				libnet.WithMasqueradeNetworking(),
```

Comment:
Does having a network connection somehow affect the test?

Reply:
No, copy-paste for this one.

Reply:
Please consider removing it.

Reply:
I'll update the PR later.